### ISSUE TITLE
Status Update: Storing Un-Viewed Updates on server reset

### ISSUE THREAD

**Issue by Pandarati** [Pandarati/CodeU-Summer-2017#13]
The storing of updates still needs to be available.

There might've been updates, but the user didn't call `status update` yet and the server resets. Thus, everything should be written to memory in this area to the LOG before the server shuts down?

**Comment by EveCoulibaly** [comment]
I agree! I am not sure how to go about doing it though. As of now the interests are stored in the model but do not have the correct update values associated with them, as new ones are created in the Chat. I don't know how to pass the number of missed messages and conversations back to the server, and how to go about writing them in the transaction LOG.

**Comment by EveCoulibaly** [comment]
I had an idea. Maybe we could have map written to the same file as the log and just write it out every so often?!

**Comment by Pandarati** [comment]
I'll handle writing them into the Transaction Log as long as long as you can get them published to the CMD screen? From the way you wrote the code, all of the Interest have their own object. You could just add a private variable that increases in the object. Then, we could have a getter method to get the count.

**Comment by Pandarati** [comment]
Well, it's not always necessary to write it out. The only case it will actually matter is if the `exit` command is called.  We want to save the unseen convo payloads to the log.
